test/client.test.js

```diff
@@ -173,14 +173,10 @@ describe('Client', () => {
   test('Client#close call on Http2Adapter-based Client with force=true', async () => {
     const client = util.getClient()
 
-    await expect(
-      Promise.all([
-        client.ping(),
-        client.close({
-          force: true,
-        }),
-      ])
-    ).rejects.toThrow(errors.ClientClosed)
+    await client.close({
```

Comment:
If guess tests should be reverted as well. In this test case, we test parallel execution of `.close` and `.ping`